p4-fusion/commands/change_list.cc

```diff
@@ -88,17 +86,15 @@ void ChangeList::StartDownload(P4API* p4, const BranchSet& branchSet, const int&
 				if (fileData.IsDownloadNeeded())
 				{
 					fileData.SetPendingDownload();
-					printBatchFiles->push_back(fileData.GetDepotFile() + "#" + fileData.GetRevision());
 					printBatchFileData->push_back(&fileData);
 
 					// Clear the batches if it fits
-					if (printBatchFiles->size() == printBatch)
+					if (printBatchFileData->size() == printBatch)
```

Comment:
```suggestion
					if (printBatchFileData->size() >= printBatch)
```

Would this be more defensive?